websites/B/bol.com/presence.ts

```diff
@@ -0,0 +1,37 @@
+const presence = new Presence({
+  clientId: "813110347165728849"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "bol",
+    buttons: [{ label: "Pagina bekijken", url: document.location.href }]
+  };
+
+  presenceData.details = "Bladert op bol.com";
+  presenceData.state = `Pagina '${document.title.replace("| ", "|").replace(" |", "|").split("|")[1]}'`;
+  if (document.location.pathname === "/" || document.location.pathname === "/nl/") presenceData.state = "Startpagina";
+
+  if (document.querySelector("span[class*=h-boxed][data-test*=title]")) {
+    presenceData.details = `Bekijkt '${document.querySelector("span[class*=h-boxed][data-test*=title]").innerHTML}'`;
+    presenceData.state = `In ${document.querySelector("ul[class*=breadcrumbs][data-test*=breadcrumb]").lastElementChild.querySelector("span[class*=breadcrumbs][data-test*=breadcrumb-name]").innerHTML}`;
+    presenceData.buttons = [{ label: "Product bekijken", url: document.location.href }];
+  } else if (document.querySelector("h1[class*=bol_header][data-test*=page-title]")) {
+    presenceData.details = `Bekijkt ${document.querySelector("h1[class*=bol_header][data-test*=page-title]").innerHTML}`;
+    presenceData.state = null;
```

Comment:
also ``delete presenceData.state`` lul

Reply:
No. It breaks otherwise.

Reply:
Why would it...

Reply:
wait how? It should work fine, you shouldn't set data to null since discord doesn't need this information! just do like I said.